packages/engine-formula/src/functions/logical/xor/index.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ErrorType } from '../../../basics/error-type';
+import type { ArrayValueObject } from '../../../engine/value-object/array-value-object';
+import { type BaseValueObject, ErrorValueObject } from '../../../engine/value-object/base-value-object';
+import { BooleanValueObject, type StringValueObject } from '../../../engine/value-object/primitive-object';
+import { BaseFunction } from '../../base-function';
+
+export class Xor extends BaseFunction {
+    override minParams = 1;
```

Comment:
`=XOR(C1:C3>0,E1:G1<10,J1:J4)` gets `TRUE`
![image](https://github.com/dream-num/univer/assets/26371161/7781cce1-fe50-4b43-a887-a9ff3ad3941a)

Reply:
@Hossein-Khan  Still need to be resolved

Reply:
Sure, could you please explain more about what the issue is with the XOR function?